apps/service/src/api/query/WorldCupQuery.ts

```diff
@@ -0,0 +1,20 @@
+import { useQuery } from "react-query";
+import { WorldCupService } from "../service";
+
+const WORLDCUP_KEY = "worldcup";
+const WITH_WHO_KEY = "with_who";
+const DRINKS_KEY = "drinks";
+
+export const useGetOneWorldCupInfoById = (worldCupId: number) =>
+  useQuery([WORLDCUP_KEY], () => WorldCupService.getOneWorldcupInfo({ worldCupId }));
```

Comment:
리액트쿼리키 값은 캐시에 매핑 되는 키 값이라 캐시를 활용 하려면 리액트쿼리키 값으로 `worldCupId` 를 같이 포함하면 좋을 것 같습니다. (다른녀석들도..)
```suggestion
  useQuery([WORLDCUP_KEY, worldCupId], () => WorldCupService.getOneWorldcupInfo({ worldCupId }));
```